src/components/cursor.js

```diff
@@ -1,94 +1,162 @@
 var registerComponent = require('../core/component').registerComponent;
+var THREE = require('../lib/three');
 var utils = require('../utils/');
 
+var EVENTS = {
+  CLICK: 'cursor-click',
```

Comment:
So you can tell which event came from which component. It emphasizes that these aren't special core events, just events provided by a component.